test/sequential/test-fs-watch.js

```diff
@@ -120,45 +120,3 @@ function repeat(fn) {
 {
   fs.watch(__filename, { persistent: false }, common.mustNotCall());
 }
-
-// Whitebox test to ensure that wrapped FSEvent is safe
-// https://github.com/joyent/node/issues/6690
-{
-  let oldhandle;
-  assert.throws(
-    () => {
-      const w = fs.watch(__filename, common.mustNotCall());
-      oldhandle = w._handle;
-      w._handle = { close: w._handle.close };
-      w.close();
-    },
-    {
-      name: 'Error',
-      code: 'ERR_INTERNAL_ASSERTION',
-      message: /^handle must be a FSEvent/,
-    }
-  );
-  oldhandle.close(); // clean up
-}
-
-{
-  let oldhandle;
-  assert.throws(
-    () => {
-      const w = fs.watch(__filename, common.mustNotCall());
-      oldhandle = w._handle;
-      const protoSymbols =
-        Object.getOwnPropertySymbols(Object.getPrototypeOf(w));
-      const kFSWatchStart =
-        protoSymbols.find((val) => val.toString() === 'Symbol(kFSWatchStart)');
-      w._handle = {};
-      w[kFSWatchStart]();
-    },
-    {
-      name: 'Error',
-      code: 'ERR_INTERNAL_ASSERTION',
-      message: /^handle must be a FSEvent/,
-    }
-  );
-  oldhandle.close(); // clean up
-}
```

Comment:
I think we want to keep those tests, possibly activating them only on Windows and macOS.

Reply:
Did the same thing. Thanks!